biz.aQute.bndlib.tests/src/test/component/DSAnnotationTest.java

```diff
@@ -2860,6 +2860,49 @@ public void testPrefixCollisionExtraAttributesDefaultPrefix() throws Exception {
 
 	}
 
+	@XMLAttribute(namespace = "override", prefix = "", embedIn = "http://www.osgi.org/xmlns/scr/*")
+	@Retention(RetentionPolicy.CLASS)
+	@Target({
+		ElementType.TYPE
+	})
+	@interface TestSpecReplace {
+		boolean immediate() default false;
+
+		boolean enabled() default true;
+	}
+
+	@TestSpecReplace
+	@Component(immediate = true, enabled = false)
```

Comment:
I don't see the point in adding a feature which will have surprising results. I say immediate=true right there in the source code, yet I don't get true because some other annotation overruled me.

I can certainly see the case where some other annotation can alter defaults but not override my explicit declaration.